src/main/java/io/jenkins/blueocean/autofavorite/FavoritingScmListener.java

```diff
@@ -93,7 +93,7 @@ public void onCheckout(Run<?, ?> build, SCM scm, FilePath workspace, TaskListene
                     first = null;
                 }
             } else {
-                LOGGER.log(Level.SEVERE, "Unexpected error when retrieving changeset", e);
+                LOGGER.log(Level.WARNING, "Unexpected error when retrieving changeset", e);
```

Comment:
Or just delete the system log message altogether, since as of #9 it should already be being printed to the build log.